aggregateur/validators.py

```diff
@@ -342,3 +342,75 @@ def schema_json_data(self):
         with open(self.filepath(self.SCHEMA_FILENAME)) as f:
             self.schema_data = json.load(f)
         return self.schema_data
+
+
+class GenericValidator(BaseValidator):
```

Comment:
If you can find a better name. Generic vs Base is not very clear. I don't have a good idea, otherwise, leave it this way.

Reply:
Generic vs Base is not ultra clear, true, but I think `generic` works well in the rest of the code. I have `meta` or  `dummy` as alternatives, not sure it's better.